herd/AArch64Sem.ml

```diff
@@ -2350,6 +2349,8 @@ module Make
                     | ADD | ADDS -> fun (v1, v2) -> M.add v1 v2
                     | EOR -> fun (v1, v2) -> M.op Op.Xor v1 v2
                     | ORR -> fun (v1, v2) -> M.op Op.Or v1 v2
+                    | ORN -> fun (v1, v2) -> M.op1 Op.Not v2
```

Comment:
The operation performed by ORN is bitwise NOT, which herd implements as `Op.Inv`.
```suggestion
                    | ORN -> fun (v1, v2) -> M.op1 Op.Inv v2
```